### PR TITLE
[Tier-2]CEPH-11148 - s3:PutBucketWebsite with users in same and different tenant

### PR DESCRIPTION
**Pull request by anrao19**
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-11148
log: http://magna002.ceph.redhat.com/ceph-qe-logs/anuchaithra/test_put_bucket_website_with_tenant_same_and_different_user.console.log

### CODE REVIEW

**Review by ckulal — COMMENTED** [review]
LGTM

File: rgw/v2/tests/s3_swift/test_bucket_policy_with_tenant_user.py [hunk 1]
```diff
@@ -118,6 +137,8 @@ def test_exec(config, ssh_con):
             action_list = ["GetObject", "GetObjectVersion"]
         elif config.test_ops.get("put_get_bucket_notification", False):
             action_list = ["GetBucketNotification", "PutBucketNotification"]
+        elif config.test_ops.get("put_bucket_website", False):
```
Comment (mkasturi18) [inline rgw/v2/tests/s3_swift/test_bucket_policy_with_tenant_user.py:140]: @anrao19 , can we avoid using elif, as we have seen issues earlier.

File: rgw/v2/tests/s3_swift/test_bucket_policy_with_tenant_user.py [hunk 2]
```diff
@@ -253,6 +274,38 @@ def test_exec(config, ssh_con):
...
+
+                log.info(
+                    "with non bucket owner of same tenant, perform put bucket wesite"
+                )
+                reusable.put_bucket_website(rgw_tenant1_user2_c, bucket.name)
+                reusable.put_bucket_website(rgw_tenant1_user3_c, bucket.name)
+
+                log.info(
+                    "with non bucket owner of different tenant, perform put bucket wesite"
+                )
```
Comment (mkasturi18) [inline rgw/v2/tests/s3_swift/test_bucket_policy_with_tenant_user.py:288]: Same as above, can we improve the log info msg

Comment (anrao19) [reply rgw/v2/tests/s3_swift/test_bucket_policy_with_tenant_user.py:288]: @mkasturi18 , made changes

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**